### DISCUSSION TITLE
Max results

### DISCUSSION THREAD

**Discussion by gmarsanich** [mattwright324/youtube-comment-suite#110] [Q&A] [answered]
How are you getting more than 100 comments per API request? I have a Python script that I adapted from the Google API page but it only retrieves 100 comments and I'm not sure how to get more than that.

**Comment by mattwright324** [comment, marked as answer]
Hi @gmarsanich 

As per the [CommentThreads: list](https://developers.google.com/youtube/v3/docs/commentThreads/list) the `maxResults` per page is 100. However, if you include the `replies` part on the request you can get up to 5 replies to each thread returned. So theoretically you could get a maximum of 100 thread comments + 500 reply comments.

If the goal is to get everything (such as with this application) you would need to still query the replies for threads where there are more than 5 replies. A large majority of threads have 5 or fewer replies so it works well to reduce the amount of calls and quota used.

**Reply by gmarsanich** [reply]
Awesome, thanks!
